terraform/aws/scripts/install.sh.tpl

```diff
@@ -3,22 +3,20 @@ set -e
 
 sudo apt-get update -y
 sudo apt-get install -y curl unzip
-curl -L "${download-url}" > /tmp/approzium.zip
```

Comment:
This was outputting a `linux_amd64.zip` file with valid contents, and an `approzium.zip` file with 0kb in it. Just needed to be tweaked to use the `-o` param for it to work as intended.